br/pkg/stream/rewrite_meta_rawkv.go

```diff
@@ -432,9 +432,21 @@ func (sr *SchemasReplace) rewriteValue(
 }
 
 // RewriteKvEntry uses to rewrite tableID/dbID in entry.key and entry.value
-func (sr *SchemasReplace) RewriteKvEntry(e *kv.Entry, cf string) (*kv.Entry, error) {
+func (sr *SchemasReplace) RewriteKvEntry(e *kv.Entry, cf string, insertDeleteRangeForTable InsertDeleteRangeForTable, insertDeleteRangeForIndex InsertDeleteRangeForIndex) (*kv.Entry, error) {
 	// skip mDDLJob
+
 	if !strings.HasPrefix(string(e.Key), "mDB") {
+		if strings.HasPrefix(string(e.Key), "mDDLJobH") { // mDDLJobHistory
+			job := &model.Job{}
+			if err := job.Decode(e.Value); err != nil {
+				log.Debug("failed to decode the job", zap.String("error", err.Error()), zap.String("job", string(e.Value)))
+				// some mddlJob kv's value is not json data, just skip it
```

Comment:
maybe explain why some mddlJob kv's value is not json data and give a clear comment here

Reply:
done.  Please take another look.